lib/termux.js

```diff
@@ -0,0 +1,28 @@
+const execa = require('execa');
```

Comment:
I think module code is always in strict mode, but I guess it doesn't hurt to add it.